etc/fibo-vue/src/views/Courses.vue

```diff
@@ -2,87 +2,169 @@
-                                    <li><b>B8.</b> Knowledge Graph Enterprise Architecture Ecosystem</li>
-                                </ul>
-                            </li>
-                            <li><b>C. FIBO Industry Model review</b> [2:00]
-                                <ul>
-                                    <li><b>C1.</b> Structure</li>
-                                    <li><b>C2.</b> Content</li>
-                                    <li><b>C3.</b> Infrastructure</li>
-                                </ul>
-                            </li>
-                            <li><b>D. Knowledge Graph Hands-on Exercises</b> [2:00]
-                                <ul>
-                                    <li><b>D1.</b> Swaps in Finance</li>
-                                    <li><b>D2.</b> Toxic byproducts in Agriculture</li>
-                                    <li><b>D3.</b> Automotive manufacturing</li>
-                                </ul>
-                            </li>
-                            <li><b>E. Knowledge Graph Course Key Takeaways</b> [0:15]
-                                <ul>
-                                    <li><b>E1.</b> Survey</li>
-                                    <li><b>E2.</b> Accreditation</li>
-                                </ul>
-                            </li>
-                        </ul>
-
-
-                        <h3><span>Course Palette</span></h3>
-                        <p><em>Open Knowledge Graph and FIBO Fundamentals Course</em> is just part of our educational offer. Our full offer of courses covers different areas of Knowledge Graph Industry. These are shown below as a mind map that interlinks different learning topics.</p>
-                        <p> </p>
-                        <br>
-                        <img style='height: 100%; width: 100%; object-fit: contain' src="../assets/img/course_palette.png" alt="FIBO Course Map" />
-                        <img style='height: 50%; width: 50%; object-fit: contain' src="../assets/img/course_palette_legend.png" alt="FIBO Course Map Legend"/>
-                        <br>
-                        <p>The map is split into six course modules, which are then split into smaller training blocks. Obviously, the modules and the blocks are not independent from one another as some require knowledge provided by others. Therefore the map shows which modules and blocks depend on which, i.e., it shows what you need to master before you can dig into something else.</p>
-                    </div>
+</h4>
+                    <p>In these challenging times EDMC will provide on <b>May 5-6</b>, <b>May 12-13</b> and <b>May 19-20</b> with&nbsp;<b><em>Open Knowledge Graph and FIBO Fundamentals Course</em></b>:</p>
+
+                    <h5>
+<span>3 class timezones, 6 hours, over 2 consecutive days course</span>
+</h5>
+<ul>
+    <li><b>AMERICAS</b></li>

```

Comment:
ante meridiem/post meridiem is not abbreviated consistently:
a and p
a.m. and p.m.